hk2-locator/pom.xml

```diff
@@ -168,5 +168,28 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>jdk9+</id>
```

Comment:
See https://github.com/apache/maven-surefire/pull/334.
To avoid such problem in the future - could other id be used?